src/shim/shim_syscall.c

```diff
@@ -78,8 +78,7 @@ bool shim_syscall(long syscall_num, long* rv, va_list args) {
                 *rv = 0;
             } else {
                 trace("found NULL timespec pointer in clock_gettime");
-                *rv = -1;
-                errno = EFAULT;
+                *rv = -EFAULT;
```

Comment:
Does this change cause an issue when preload mode is used without seccomp (does the errno still need to be set somewhere)?